sections/02_intro.md

```diff
@@ -34,41 +34,88 @@ learning methods more challenging or less fruitful to apply.
-employ networks of limited depth. We found it valuable to include these as we
-sought to identify the current contributions of deep learning and to predict its
-future impact. Researchers may employ these shallow learning methods for a
-number of reasons including: 1) shallow networks provide a degree of
-interpretability that better matches their use case; 2) the available data are
-insufficient to support deeper architectures, however new datasets that will
-support deep methods are expected; 3) or as building blocks to be combined with
-other non-neural-network-based approaches at subsequent stages.
+[This section needs citations]
+
+Deep Learning is a collection of new techniques that together have recently demonstrated 
+breakthrough gains over existing approaches in several fields. 
+
+Deep learning is built on a very old idea, neural networks, that was first
+proposed in 1943 [doi:10.1007/BF02478259] as a model for how biological
+brains proces information. Since then, interest in neural networks a computational
+models has waxed and waned several times. This history is interesting in its own right  [@doi:10.1103/RevModPhys.34.135, @doi:10.1103/RevModPhys.34.135],
+but in recent years, with the advances of Deep Learning, attention has shifted back.
+
+Several important advances make the current surge of work done in this area possible.
+
+First, several easy to use software packages (Tensorflow, Caffe, Theano) now enable a much broader range of scientists
+to build and train complicated. In the past, neural networks required very specialized knoweldge to 
+build and modify, including the ability to robustly code differentials of matrix 
+expressions. Errors here are often subtle and difficult to detect, so it could be 
+very difffult to tailor networks to specific problems without substantial experience
+and training. Now, however, with these new packages, even very complex neural networks 
+are automatically differentiated, and high level scripting instructions can transparently
+run very efficently on GPUs. The technology has progressed to the point that even
+algorithms can be differentiated [cite neural stack and neural memory papers].
+
+Second, key technical insight has been uncovered that guides the construction of 
+much more complicated networks that previously possible. In the past, most neural networks 
+included just a single network. A network with an arbitrary number of hidden nodes, but 
+just a single layer, can learn arbitrarily complex functions. And networks with more than one hidden
+layer (deep networks), were hard to train. However, it turns out, deep networks can more
+efficiently represent many tasks when they are built to mirror the underlying structure of the data.
+Moroever, deep networks are more robust and trainable when employing several
+architectural innovations: weight replication, better behaived non-linearities like rectified-linear units, residual networks, 
```

Comment:
These are all very good to point out, but I wonder whether our readers will have any idea what these mean or if they will be unintuitive jargon?